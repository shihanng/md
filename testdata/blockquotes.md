Before

> First line
> Another line
>
> Second paragraph first line
> Second paragraph another line
>
> > Nested line
>
> Last line

After
