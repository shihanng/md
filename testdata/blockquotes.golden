Before

> > First line Another line
>
> Second paragraph first line Second paragraph another line\
> Second paragraph last line
>
> > Nested line\
> > Third paragraph
>
> Last line\

After
